core/integer/even_spec.rb

```diff
@@ -9,10 +9,12 @@
     1.even?.should be_false
     2.even?.should be_true
 
```

Comment:
This case for instance, would nicely be split in
`Integer#even? for Fixnum returns ...` and
`Integer#even? for Bignum returns ...`.